apps/system/js/notifications.js

```diff
@@ -460,6 +464,17 @@ var NotificationScreen = {
     messageContent.setAttribute('dir', 'auto');
     message.appendChild(messageContent);
     notificationNode.appendChild(message);
+    var countDiv = document.createElement('div');
+    countDiv.classList.add('detail-count');
```

Comment:
As per UX, if a particular notification has not focus a generic message will be displayed like "2 New Messages" and "3 Missed Calls" else messages clubbed in a ',' separated string.